core/src/org/pentaho/di/core/database/DataSourceProviderInterface.java

```diff
@@ -38,6 +38,19 @@
    * @param datasourceName
    * @return javax.sql.DataSource
    */
-  public DataSource getNamedDataSource( String datasourceName ) throws DataSourceNamingException;
```

Comment:
Sorry, I overlooked the fact this was a method, thought it was changing the interface itself to package private.